src/macros.jl

```diff
@@ -60,8 +60,13 @@ macro polyvariable(args...)
   end
 
   # separate out keyword arguments
-  kwargs = filter(ex->isexpr(ex,:kw), extra)
-  extra = filter(ex->!isexpr(ex,:kw), extra)
+  if VERSION < v"0.6-"
```

Comment:
We will need to be more specific about the version cutoff like what is done here: https://github.com/JuliaOpt/JuMP.jl/commit/b8a321e2862179a00087e65ab80414b49e79d714
However I have no idea how to find the number in the version v"0.6.0-dev.$number". @tkelman How can we find it ?